src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.types.js

```diff
@@ -5,6 +5,7 @@ export type Props = {|
     program: Program,
     enrollmentId: string,
     teiId: string,
+    events: any,
```

Comment:
Hey, 
Why did you use `any` here and in the rest of the PR? Don't we know the shape of the `events` array? In my opinion `any` should be reserved for very complex situations. 
Thanks!

Reply:
Hey, sorry I sow too late your comment! I will delete my comments. Thanks!